src/sora_track_interface.h

```diff
@@ -39,6 +42,20 @@ class SoraTrackInterface : public DisposePublisher, public DisposeSubscriber {
     return track_->state();
   }
 
+  /**
+   * この Track の Stream ID を std::string で返します。
+   * 
+   * Python で呼び出すための関数です。
+   * 本来 Track には複数の Stream ID を紐づけることができるのですが、
+   * Sora の使用上 Track には Stream ID が 1 つしか紐づかないため Track のメンバーとしました。
+   * 
+   * このクラスを継承している SoraAudioSource, SoraVideoSource については、
+   * Connection にセットしてある際にセットされた組み合わせで Connection ごとに Stream ID が新規に設定されること。
+   * SoraAudioSource, SoraVideoSource は複数の Connection に割り当てが可能なこと。
```

Comment:
まだ文の途中なので行末の `。` は不要そうに見えました。
```suggestion
   * SoraAudioSource, SoraVideoSource は複数の Connection に割り当てが可能なこと
```